services/github.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from environs import Env
+from furl import furl
+
+
+env = Env()
+
+
+BASE_URL = "https://api.github.com"
+GITHUB_TOKEN = env.str("GITHUB_TOKEN")
+
+
+session = requests.Session()
+session.headers = {
+    "Accept": "application/vnd.github.v3+json",
+    "Authorization": f"bearer {GITHUB_TOKEN}",
+    "User-Agent": "OpenSAFELY Interactive",
+}
+
+
+def create_issue(analysis_request_id):
+    body = f"""
+    Workspace: https://jobs.opensafely.org/datalab/opensafely-interactive/opensafely-interactive/
```

Comment:
Technically, we could construct this URL rather than hardcoding it. We're doing that elsewhere, so it would be more consistent.